packages/frontend/src/components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -27,61 +25,19 @@ const ConnectWalletModal: FunctionComponent<ConnectWalletModalProps> = ({
   }, [error]);
 
   return (
-    <Transition appear show={isOpen} as={Fragment}>
-      <Dialog as="div" className="relative z-10" onClose={closeModal}>
-        <Transition.Child
-          as={Fragment}
-          enter="ease-out duration-300"
-          enterFrom="opacity-0"
-          enterTo="opacity-100"
-          leave="ease-in duration-200"
-          leaveFrom="opacity-100"
-          leaveTo="opacity-0"
-        >
-          <div className="bg-new-black fixed inset-0 bg-opacity-60" />
-        </Transition.Child>
-
-        <div className="fixed inset-0 overflow-y-auto">
-          <div className="flex min-h-full items-center justify-center p-4 text-center">
-            <Transition.Child
-              as={Fragment}
-              enter="ease-out duration-300"
-              enterFrom="opacity-0 scale-95"
-              enterTo="opacity-100 scale-100"
-              leave="ease-in duration-200"
-              leaveFrom="opacity-100 scale-100"
-              leaveTo="opacity-0 scale-95"
-            >
-              <Dialog.Panel className="bg-new-black border-flashbang-white font-display w-full max-w-md transform overflow-hidden rounded-md border border-b-[6px] p-10 pb-4 align-middle shadow-xl transition-all">
-                <Dialog.Title
-                  as="h3"
-                  className="bg-new-black text-flashbang-white border-b-darker-gray mb-7 max-w-md border-b-[1px] pb-5 text-left text-4xl font-medium uppercase"
-                >
-                  Connect <br /> your wallet
-                </Dialog.Title>
-                {supportedWallets.map(wallet => (
-                  <WalletOption
-                    // Todo(Nathan): Make this dynamic
-                    injectedWalletName={wallet === 'injected' ? 'MetaMask' : undefined}
-                    key={wallet}
-                    walletOption={wallet}
-                    activate={() => connectWallet(wallet)}
-                  />
-                ))}
-                <button
-                  aria-label="Close dialog"
-                  className="absolute top-10 right-10"
-                  onClick={closeModal}
-                  type="button"
-                >
-                  <img src={closeIcon} alt="Close icon" />
-                </button>
-              </Dialog.Panel>
-            </Transition.Child>
-          </div>
-        </div>
-      </Dialog>
-    </Transition>
+    <Modal isOpen={isOpen} handleClose={closeModal} title="Connect your wallet">
+      <div className="pb-6">
+        {supportedWallets.map(wallet => (
+          <WalletOption
+            // Todo(Nathan): Make this dynamic
```

Comment:
By dynamic, do you mean using "Brave Wallet" / "MetaMask" etc.

Reply:
This is an old comment actually, but yeah.